lib/ansible/parsing/vault/__init__.py

```diff
@@ -1262,11 +1262,18 @@ def _encrypt_pycrypto(b_plaintext, b_key1, b_key2, b_iv):
 
         return to_bytes(hmac.hexdigest(), errors='surrogate_or_strict'), hexlify(b_ciphertext)
 
+    @classmethod
+    def _get_salt(cls):
+        custom_salt = os.getenv('ANSIBLE_VAULT_SALT')
```

Comment:
this is no tthe correct way to add configuration, use lib/ansible/config/base.yml entry

Reply:
Also, add a changelog entry and integration test for this change.

Reply:
I've added `DEFAULT_VAULT_ENCRYPT_SALT` to configuration, also changed name of envvar to `ANSIBLE_VAULT_ENCRYPT_SALT` to make it more clear.